### DISCUSSION TITLE
Option to use static ports in launchSettings.json

### DISCUSSION THREAD

**Discussion by glen-84** [dotnet/aspnetcore#57513] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

When writing documentation, for example, it's much easier to just say "open localhost:5000", rather than "open the localhost URL, the port will vary from 5000 to 5300 ...".

Also, in many cases, developers are not running multiple applications at the same time.

### Describe the solution you'd like

I understand why you may want to keep the current default behaviour, but a CLI option (to `dotnet new`) would be useful.

**Comment by gfoidl** [comment]
Just for completeness: the `applicationUrl` in the `launchSettings.json` file can be adjusted in order to use the same port number.

**Reply by glen-84** [reply]
This means that you have to [instruct the reader](https://github.com/ChilliCream/graphql-workshop/blob/58292123e38dfdb7a99f0cb53c34201ac05a9e60/docs/1-creating-a-graphql-server-project.md#L17-L20) to perform these changes.
